docs/pages/textarea.js

```diff
@@ -53,6 +53,265 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('https://www.pinterest.com/pin/768145280205600341/');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-dont-single-line"
+        label="Destination URL"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use `label` to clearly denote what information the user should input. Use `placeholder` sparingly as [they can erode usability of form fields](https://www.nngroup.com/articles/form-design-placeholders/)."
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-do-label"
+        label="Tell everyone what this Pin is about"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Remove the label, as this creates accessibility and usability issues."
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-dont-placeholder"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+        placeholder="Tell us your story"
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use helperText to provide additional context that will aid the user in most effectively inputing information."
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-do-helpertext"
+        label="Explain what people can see in this Pin"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+        helperText="This text will be read aloud by screen readers"
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use placeholder to provide any information necessary to filling out the form field. Placeholder text disappears after the user begins entering data and should not contain crucial information."
```

Comment:
nit: either `placeholder` should have backticks (if referring to the prop), or an article ("the placeholder") if referring to placeholders in general

Reply:
Done